tests/sentry/feedback/usecases/test_create_feedback.py

```diff
@@ -768,3 +770,47 @@ def test_create_feedback_spam_detection_set_status_ignored(
         group = Group.objects.get()
         assert group.status == GroupStatus.IGNORED
         assert group.substatus == GroupSubStatus.FOREVER
+
+
+# Unit tests for shim_to_feedback error cases. The typical behavior of this function is tested in
+# test_project_user_reports, test_post_process, and test_update_user_reports.
+
+
+@django_db_all
+def test_shim_to_feedback_missing_event(default_project, monkeypatch):
+    # Not allowing this since creating feedbacks with no environment doesn't work well.
+    mock_create_feedback_issue = Mock()
+    monkeypatch.setattr(
+        "sentry.feedback.usecases.create_feedback.create_feedback_issue", mock_create_feedback_issue
+    )
+    report_dict = {
+        "name": "andrew",
+        "email": "aliu@example.com",
+        "comments": "Shim this",
+        "event_id": "a" * 32,
+        "level": "error",
+    }
+    shim_to_feedback(
```

Comment:
looks like the tests caught this but i think the `shim_to_feedback` function requires an `event` to be passed in -- might need to fix the typing?

Reply:
Hm weird it's warning about type but not in the test below. I guess we could remove both, since they both test params different from the annotated types.

Reply:
What do you think? @michellewzhang

Reply:
discussed offline - we're going to try to ignore `mypy`, since we want to keep the tests to assert that the behavior is as expected when the stuff passed in is junk